Source/WebCore/editing/CharacterRange.h

```diff
@@ -46,6 +46,11 @@ struct CharacterRange {
     CharacterRange() = default;
     constexpr CharacterRange(uint64_t location, uint64_t length);
 
+    bool operator==(const CharacterRange& range) const
```

Comment:
I think you can use `= default` here.

Reply:
I have no idea how to format that, and searching our codebase isn't helping...

Reply:
`bool operator==(const CharacterRange& range) const = default;`